src/test/java/bio/terra/common/fixtures/JsonLoader.java

```diff
@@ -30,6 +31,11 @@ public String loadJson(final String resourcePath) throws IOException {
         return objectMapper.readerFor(resourceClass).readValue(json);
     }
 
+    public <T> T loadObject(final String resourcePath, final TypeReference<T> typeReference) throws IOException {
```

Comment:
I may be crazy, but I think you can replace both of these `loadObject()` methods with a single method that infers its return type from its context. Something like this should work:
```
    public <T> T loadObject(String resourcePath) throws IOException {
        return objectMapper.readValue(loadJson(resourcePath), new TypeReference<T>() { });
    }
```